be/src/common/config.h

```diff
@@ -690,6 +690,11 @@ CONF_mInt64(experimental_s3_min_upload_part_size, "16777216");
 CONF_Int64(max_load_dop, "16");
 
 CONF_Int64(meta_threshold_to_manual_compact, "10737418240"); // 10G
+// size of grf generated by broadcast join below this limit, multiple rf copy will be delivered in passthrough
+// style, otherwise, rf will be relayed by other be.
+CONF_Int64(deliver_broadcast_rf_passthrough_bytes_limit, "131072");
```

Comment:
It seems config:deliver_broadcast_rf_passthrough_bytes_limit unused in this PR.

Reply:
used in RuntimeFilterPort::publish_runtime_filters and RuntimeFilterWorker::_process_send_broadcast_runtime_filter_event